deployments/docker/ai-unlimited.yaml

```diff
@@ -6,20 +6,38 @@ services:
       replicas: 1
     platform: linux/amd64
     container_name: ai-unlimited-workspaces
-    image: ${AI_UNLIMITED_IMAGE_NAME:-teradata/ai-unlimited-workspaces}:${AI_UNLIMITED_IMAGE_TAG:-latest}
+    image: ${AI_UNLIMITED_IMAGE_NAME:-teradata/ai-unlimited-workspaces}:${AI_UNLIMITED_IMAGE_TAG:-v0.2.23}
```

Comment:
Because this is adding the new workspaces UI, I think the compatible versions of workspaces will be `v0.2.24` and up (also `v0.3.0`)

`v0.2.23` is the old workspaces that didn't come with the new UI.